.changeset/825aeca5/changes.md

```diff
@@ -0,0 +1 @@
+- Updating the template to match the changes in the todo demo app
```

Comment:
Not really something that's worth updating since the changelogs aren't hugely relevant right now since ks-5 isn't widely used but this comment feels more like a commit message rather than a changelog entry since you can't understand part of the changes that happened to the package unless you have more context surrounding this change. e.g. If I was just a consumer of this package, I would think, what were the changes in the todo demo app?